<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVisualization</class>
 <widget class="QDialog" name="DialogVisualization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1274</width>
    <height>1012</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vizualization</string>
  </property>
  <widget class="QLabel" name="lPanelVisualization">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>1021</width>
     <height>640</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_rough">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>60</y>
     <width>102</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rough data</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_smooth">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Filtered data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_cancel">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_play">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelXY">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>720</y>
     <width>1021</width>
     <height>211</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelFixations">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_14">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>710</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lsecsSegment">
   <property name="geometry">
    <rect>
     <x>1060</x>
     <y>720</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelXY_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>670</y>
     <width>1021</width>
     <height>261</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lPanelMissingData">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>930</y>
     <width>1021</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_15">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>670</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fixations</string>
   </property>
  </widget>
  <widget class="QLabel" name="lLabel_16">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>720</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gaze Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="lProcess">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>370</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPupilDilation">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>260</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Show pupil dilation</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>30</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Data type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>190</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Additional options:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbFixations">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Paint fixations</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rad_rough_smooth">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>120</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Rough + Filtered data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
